src/test/java/org/opensearch/security/test/helper/rest/RestHelper.java

```diff
@@ -92,8 +92,6 @@
 import org.opensearch.security.test.helper.cluster.ClusterInfo;
 import org.opensearch.security.test.helper.file.FileHelper;
 
-import static org.junit.jupiter.api.Assertions.fail;
```

Comment:
I was getting missing class exception when this assertion was triggered at runtime.